mage/algorithms/traditional-graph-analytics/degree-centrality.md

```diff
@@ -0,0 +1,48 @@
+---
+id: degree-centrality-algorithm
+title: Degree Centrality
+sidebar_label: Degree Centrality
+---
+
+## Description
+
+**Degree Centrality** is a method for estimating the centrality of graph 
+entities.
+It is a simple centrality measure that counts how many neighbors a node has. 
+For a directed graph, we have two versions of the measure: in-degree and 
+out-degree.
+In-degree counts the number of in-coming edges, and out-degree counts the number
+of out-going edges.
+
+Usually, applications are more interested in in-degree measure, since in-coming
+edges are given by other nodes in a graph. Degree centrality can be used for any
```

Comment:
```suggestion
edges are defined by other nodes in a graph. Degree centrality can be used for any
```

"given" sounds very strange... what would it mean?

Reply:
For instance, in a social network, a directed edge from me to you would mean I consider you a friend, without it meaning you consider me a friend. So for some analysis of your social influence, it would be better to look at how many people consider you a friend than it is to look at how many people you consider a friend. (Not the greatest example, but what I'm getting at is that incoming edges would prove you have a social influence more than outgoing edges, hence degree centrality of your node)

Reply:
wrote it differently